sdk/nodejs/proto/alias_pb.d.ts

```diff
@@ -0,0 +1,91 @@
+// package: pulumirpc
+// file: pulumi/alias.proto
+
+import * as jspb from "google-protobuf";
+
+export class Alias extends jspb.Message {
```

Comment:
I wonder if exported things in these `d.ts` files will start showing up in our [core SDK docs](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/pulumi/), which I'd prefer not to happen, since these are really meant for internal use (aside from `@pulumi/policy`, but that usage is also really only for us).

If I have a spare moment, I'll try running https://github.com/pulumi/docs/blob/master/scripts/run_typedoc.sh with these changes to see.

If these do show up in the docs, we might have to modify our tool to exclude them, or post-process these files and add `/** @internal */` to any line that's exporting things. The former seems ideal offhand, but there are plans to move us off our custom tool and on to a standard doc tool/template for generated TS docs, so we may have to go with the latter.

Reply:
I might be running the tool wrong, but it doesn't look like these are getting picked up by that. I think because it reads tsconfig and we're not adding these to the "files" in that.